libs/deal-ticket/src/components/deal-ticket/deal-ticket.tsx

```diff
@@ -158,6 +158,12 @@ export const DealTicket = ({
     return disabled;
   }, [order]);
 
+  useEffect(() => {
```

Comment:
chore(blocking): we remove this effect and instead move this logic to the TIF callback function.

Reply:
I did this initially but it's not enough. 
Whenever it gets disabled we need to make sure postOnly gets false

Reply:
updated again reduceOnly